integration/deploy/compose_test.go

```diff
@@ -69,7 +69,9 @@ const (
       retries: 5
       start_period: 30s
   db:
-    image: alpine
```

Comment:
with `alpine` the init job would exit with an error, we never noticed because the deploy didn't wait